src/Kis/Time.hs

```diff
@@ -0,0 +1,21 @@
+module Kis.Time
+   ( TimeOffset(..)
+   , KisTime(..)
+   )
+where
+
+import Data.Time.Clock
+
+newtype TimeOffset = TimeOffset DiffTime
+
+-- | KisTime indicates wheter KIS is running in real time,
+-- or in virtual time (for simulation). If running in Virtual
+-- time the time multiplier and starting point of the simulation
+-- are in the Type aswell
+data KisTime = RealTime | VirtualTime UTCTime Int
```

Comment:
Ja vielleicht ist das einfacher, kann ich so machen. Für das Ändern des Multiplier sehe ich keinen Unterschied, man müsste in beiden Fällen den State updaten und da kann man auch eine neue Instanz von `Clock` generieren.